zephyr/include/arch/string.h

```diff
@@ -43,8 +43,10 @@ static inline int arch_memcpy_s(void *dest, size_t dest_size,
 	if (!dest || !src)
 		return -EINVAL;
 
```

Comment:
Please add the warning message inside the commit mesage. Also, isn't this code supposed to be part of initial submit?

Reply:
@dbaluta this is just a poc exercise, it is not meant for master, at least not for now.

Reply:
@Vudentz all right :)